### PR TITLE
Convert build to create-react-app

### PR DESCRIPTION
**Pull request by hebaishi**
Adds a new build using `create-react-app`. As far as I can see, this build is an improvement from the previous approach in a number of ways:
1. We get a hot-reloading development server, which proxies results to the backend.
2. We get a properly working, optimized production build
3. We get linting. I can see already that the linting has identified a number of issues, mainly missing variables and these have now been fixed.
So, to business:
`create-react-app` is a tool used for bootstrapping a React project, and provides extremely convenient build targets. It also performs linting. I have used yarn as the package manager here, so you will need to install it. It's a very popular node package manger, and in my opinion, far better than npm. You can get it [here](https://classic.yarnpkg.com/en/docs/install/#debian-stable). Just install the latest stable version. That'll be fine.

Then, after you have run:
```
ddev start
```
To get started with the new build, you need to do:
```
cd pocovidscreen/resources/ui
yarn install
yarn run start
```
For a production build, you can do:
```
yarn run build
```
@stat1x I have reverted one of the changes you made recently, where you deleted the public folder.
Unfortunately, we are going to need it. Removing it will break the build, as the build looks for an `index.html` file in that location to use as an entry point for the app. Also, I've removed all files from the `web_root` folder. We should keep that folder empty because it's meant for generated files only. Mixing generated files and source files in the same folder only causes confusion, so I would much prefer it if we kept the `web_root` folder empty and just used it as a location for putting the generated files.

### CODE REVIEW

File: .github/workflows/yarn_build.yml [hunk 1]
```diff
@@ -16,8 +16,8 @@ jobs:
     - name: Build
       run: |
         cd pocovidscreen/resources/ui
-        npm install
-        npm run prod
+        yarn install
+        yarn run build
```
Comment (jannisborn) [inline .github/workflows/yarn_build.yml:20]: If we migrate from `npm` to `yarn`, we might want to adapt the `README.md` to ease reproduction.

File: pocovidscreen/resources/ui/src/components/Slider/Slider.js [hunk 2]
```diff
@@ -6,15 +6,15 @@ import chevronLeftLight from './chevron-left.svg';
...
-import saheli from '../../../assets/images/team/saheli.jpg';
+import julie from '../../images/team/julie.jpg';
+import marion from '../../images/team/marion.jpg';
+import gabriel from '../../images/team/gabriel.jpg';
+import manuel from '../../images/team/manuel.jpg';
+import nina from '../../images/team/nina.jpg';
+import janis from '../../images/team/janis.jpg';
+import jay from '../../images/team/jay.jpg';
+import lorenzo from '../../images/team/lorenzo.jpg';
+import saheli from '../../images/team/saheli.jpg';
```
Comment (jannisborn) [inline pocovidscreen/resources/ui/src/components/Slider/Slider.js:17]: @hebaishi you should be there too ;)

Comment (stat1x) [reply pocovidscreen/resources/ui/src/components/Slider/Slider.js:17]: Sure, we need to add you here @hebaishi.

File: .github/workflows/yarn_build.yml (hunk 1 above)
Comment (hebaishi) [reply .github/workflows/yarn_build.yml:20]: Done :+1:

**Pull request closed without merging**